### ISSUE TITLE
Cannot add a new gesture binding for "Entry point for NVDA Dev & Test Toolbox layered commands".

### ISSUE THREAD

**Issue by hwf1324** [CyrilleB79/NVDA-Dev-Test-Toolbox#24]
## Steps to reproduce:

Add a new gesture binding for "Entry point for NVDA Dev & Test Toolbox layered commands", for example `NVDA+alt+shift+x`. Then press Enter to confirm.

## Actual behavior:

Pressing the bound gesture in input help mode does not report the bound script.

## Expected behavior:

A gesture can be bound to this script.

## Additional information:

NVDA Version: alpha-52461,e6557a72 (2026.1.0.52461)

ndtt Version: 7.2

https://github.com/CyrilleB79/NVDA-Dev-Test-Toolbox/blame/5a3df5f39b2223cf0205b58ac03101227eb8bc48/addon/globalPlugins/ndtt/layeredGestures.py#L87C24-L87C24

**Comment by CyrilleB79** [comment]
Thanks for reporting.
The NVDA version you are using is not compatible with the add-on.
Anyway, I have also confirmed the issue on my end with a compatible NVDA version, namely 2025.3beta3, so I'll take care to fix this when I can.
